lightning/src/offers/invoice.rs

```diff
@@ -677,6 +677,10 @@ macro_rules! invoice_accessors { ($self: ident, $contents: expr) => {
 	pub fn signing_pubkey(&$self) -> PublicKey {
 		$contents.signing_pubkey()
 	}
+
+	pub(crate) fn tlv_stream(&$self) -> PartialInvoiceTlvStreamRef {
```

Comment:
Instead of defining this, can't we use `Bolt12Invoice::as_tlv_stream` by removing `#[cfg(test)]` and making it `pub(crate)`? I think it's fine to use `FullInvoiceTlvStreamRef` even though it includes the signature.